package.json

```diff
@@ -66,9 +66,9 @@
 		"prern-bundle": "patch-package --patch-dir gutenberg/packages/react-native-editor/metro-patch",
 		"rn-bundle": "react-native bundle",
 		"postrn-bundle": "patch-package --reverse --patch-dir gutenberg/packages/react-native-editor/metro-patch",
-		"i18n:check-cache": "./bin/i18n-check-cache.sh jetpack wp-plugins/jetpack layout-grid wp-plugins/layout-grid",
-		"i18n:update": "./bin/i18n-update.sh jetpack wp-plugins/jetpack ./jetpack/projects/plugins/jetpack/extensions layout-grid wp-plugins/layout-grid ./block-experiments/blocks/layout-grid",
-		"i18n:update:test": "./bin/i18n-update.sh --path i18n-test jetpack wp-plugins/jetpack ./jetpack/projects/plugins/jetpack/extensions layout-grid wp-plugins/layout-grid ./block-experiments/blocks/layout-grid",
+		"i18n:check-cache": "./bin/i18n-check-cache.sh jetpack wp-plugins/jetpack layout-grid wp-plugins/layout-grid jetpack-videopress-pkg wp-plugins/jetpack",
```

Comment:
Note that the VideoPress translations are fetched from the GlotPress project of the Jetpack plugin (internal discussion: p1680270555812159-slack-C03TA48NSUX).